### PR TITLE
feat(fido2): add fido2 support

### PR DESCRIPTION
**Pull request by imskr**
## Because

**Add Fido2 webauthn support**

## This commit

**Add fido2 libraries and helpers**

## Issue that this pull request solves

Closes: #5548 #5579 #5580 #5581 #5582 

**Added following attestations:**

- [X] Fido-U2F
- [X] Fido-Packed

## Checklist

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests that pass locally.
- [ ] I have added necessary documentation (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of a change in the user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Comment by imskr** [comment]
@vbudhram A great article on packed-attestation on [medium](https://medium.com/@herrjemand/verifying-fido2-packed-attestation-a067a9b2facd)

File: packages/fxa-auth-server/lib/fido2/keys/Packed.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    basicConstraints,
+  };
+};
+
+/**
+ * Parse attestation auth data
+ * @param  {Buffer} buffer - authData buffer
+ * @return  {Object}          - parsed authData struct
+ */
+const parseAttestation = (buffer) => {
```
[the remark this answers is not in the document]
Comment (imskr) [reply packages/fxa-auth-server/lib/fido2/keys/Packed.js:47]: Okay so, do I have to remove some of those?

File: packages/fxa-auth-server/lib/fido2/keys/Packed.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
+const jsrsasign = require('jsrsasign');
```
[the remark this answers is not in the document]
Comment (imskr) [reply packages/fxa-auth-server/lib/fido2/keys/Packed.js:1]: Okay I will change it.

File: packages/fxa-auth-server/lib/fido2/keys/Packed.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vbudhram) [reply packages/fxa-auth-server/lib/fido2/keys/Packed.js]: @imskr No you don't need to remove the current documentation, but you can create a new issue to fill in any missing documentation.

[the remark this answers is not in the document]
Comment (imskr) [reply packages/fxa-auth-server/lib/fido2/keys/Packed.js]: Okay

**Pull request merged**